### ISSUE TITLE
bird: > v1.6.5 update

### ISSUE THREAD

**Issue by george-angel** [projectcalico/calico#2470]
Calico: v3.5.0

We are currently working on cross-cluster configuration between two clusters both running calico. Our pod communication is working and we are happy with the setup.

To enable Kubernetes Service <-> Service communication we need calico nodes bgp agents (bird) to set LLGR (2 minutes). More information can be found here: https://vincent.bernat.ch/en/blog/2018-bgp-llgr

Currently the routing node takes 2 minutes to update, due to GR setting. This means `Service` requests fail for 2 minutes. Setting `llgr = 2m` and setting `gr = 0m`, should preserve the guards against flapping, but also enable quick failover for single routing nodes.

Slack thread discussing this: https://calicousers.slack.com/archives/C0BCA117T/p1551353214002600

Thank you.

**Comment by caseydavenport** [comment]
Sorry for the delay - this sounds like something that would be good to do.

We'll need to merge in the latest BIRD with our changes and do some rigorous testing of it as a first step.

**Comment by neiljerram** [comment]
@george-angel FYI, I'm working now on updating our BIRD fork to upstream v1.6.7.  (https://github.com/projectcalico/node/pull/314)

Please can I check with you: once that is in place, are you also asking/suggesting that calico-node **by default** emits specific LLGR and GR settings for all of its BGP peering configs?  Or do you already have some workflow of your own for overriding our BIRD config, and so you only need the upgraded BIRD **code**?

**Comment by george-angel** [comment]
@neiljerram great news.

We do not have any workflow *yet*. But when we started working on this, we thought we could conceivably overwrite the config with the values I mentioned. We then realised our config had no effect since the version was too old and didn't support the settings.

So if you chose to include those by default - great, less custom code on our side. If not, we would have an init-container of some sort to load the config.
